src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -449,7 +449,7 @@ static bool mergeAttrInfo(const BSONObj& attr, ContextAttribute* caP, BSONObj* m
    *    'copied' from DB to the variable 'ab' and sent back to mongo, to not destroy the value  */
   if (caP->valueType != orion::ValueTypeNotGiven)
   {
-    caP->valueBson(ab);
+    caP->valueBson(ab, getStringFieldF(attr, ENT_ATTRS_TYPE), ngsiv1Autocast && apiVersion == V1);
```

Comment:
Style guide:
... , ngsiv1Autocast && (apiVersion == V1))

Reply:
Fixed in f65e5f29a1e53753c25d1c2e81ea7e3277fb241c